demos/flask/app.conf

```diff
@@ -0,0 +1,3 @@
+[app]
+
+skey = e806cf5156fd85b2f384a1181d12e5c9e833f15d
```

Comment:
We should remove this information. Also, if this is a Duo `skey` then you'll want to get rid of that integration, since this is basically the same as putting your plaintext password out on the Internet.